relayer_api/services/database/databaseManager.js

```diff
@@ -0,0 +1,24 @@
+const CounterfactualData = require('./models/counterfactualData')
+
+module.exports = {
+
+    addToDatabase: async function(counterfactualContractAddress, fullCounterfactualData){
```

Comment:
I feel these should be much more meaningful in the context - `saveCounterfactualData`, `findCounterfactualData`, etc...